src/pages/New.vue

```diff
@@ -1,12 +1,131 @@
 <template>
   <q-page padding>
-    <h1>New Meme Page</h1>
+    <h6>Upload Meme</h6>
+    <form @submit.prevent.stop="onSubmit">
+      <q-input
+        ref="memeTitle"
+        v-model="memeTitle"
+        outlined
+        label="Enter a title..."
+        :rules="[val => !!val || 'Field is required']"
+      />
+      <div id="dropbox">
+        <q-input
+          id="new-meme-input"
+          v-model="fileInput"
+          outlined
+          type="file"
+          accept="image/*"
+          clearable
+          @change="fileChosen"
+        />
+        <q-input
+          id="new-meme-facade"
+          ref="fileName"
+          v-model="fileName"
+          outlined
+          label="Choose file..."
+          :rules="[val => !!val || 'Field is required']"
+        />
+      </div>
+      <q-item>
+        <img id="new-meme-preview">
+      </q-item>
+      <q-btn
+        label="Upload"
+        color="primary"
+        type="submit"
+        class="full-width"
+      />
+    </form>
   </q-page>
 </template>
 
 <script>
+// eslint-disable-next-line
+const CLOUD_NAME = 'kingwizard'; // TODO: Channge to correct cloud
+const CLOUD_PRESET = 'h4c8kn5g'; // TODO: Add upload preset to correct cloud
```

Comment:
Add `CLOUDINARY_NAME` and `CLOUDINARY_PRESET` to `.example.env` and talk about it in the readme.  Keep secrets hidden and out of git, even though I think we're sending it to the client :P

```
const CLOUD_NAME = process.env.CLOUDINARY_NAME
const CLOUD_PRESET = process.env.CLOUDINARY_PRESET
```